app.js

```diff
@@ -1,38 +1,80 @@
+const inputLabel = document.getElementById('inputLabel');
 const userInput = document.getElementById('todoInput');
 const addBtn = document.querySelector('#addButton');
 const list = document.querySelector('#list');
 
-let listItem = document.createElement('li');
```

Comment:
How come the above are consts and the below vars are let? let allows you to declare variables that are limited to the scope of a block statement or method.